### ISSUE TITLE
Composer Timeout Error

### ISSUE THREAD

**Issue by Suba-IC** [jvlcode/laravel-workshop#1]
Hi,
I am stuck with the below error and I could not find any solutions in any forums. Can you help me with this? 

Creating a "laravel/laravel" project at "./student-registration"
Installing laravel/laravel (v11.0.5)
  - Downloading laravel/laravel (v11.0.5)
A connection timeout was encountered. If you intend to run Composer without connecting to the internet, run the command again prefixed with COMPOSER_DISABLE_NETWORK=1 to make Composer run in offline mode.
  - Downloading laravel/laravel (v11.0.5)
  - Downloading laravel/laravel (v11.0.5)
  - Downloading laravel/laravel (v11.0.5)
    Failed to download laravel/laravel from dist: curl error 28 while downloading https://api.github.com/repos/laravel/laravel/zipball/58baff2c70144c7cd4c04eecb51fee5a24e873a8: Resolving timed out after 10001 milliseconds
    Now trying to download from source

In GitDownloader.php line 82:

  git was not found in your PATH, skipping source download

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package> [<directory> [<version>]]]

**Comment by jvlcode** [comment]
Its because you need enable some PHP extensions. I mentioned on the video I
think.

Just go to your "xampp" folder
Open "php" folder
Open php.ini file in any editor
Search for  ";extension=zip"

Remove ; symbol in the start and save file
Try again the installation it would  work

**Comment by Suba-IC** [comment]
Yes, I have already done that but it still continues to say Timeout error. 
Also I tried increasing the time thru this below code
composer config --global process-timeout 2000

**Comment by Suba-IC** [comment]
Hi, I could not find any other forum solving this issue. And I tried in some other ways also. But still I receive the same issue. Can you please guide and help me in this.
